source-jira-native/source_jira_native/resources.py

```diff
@@ -185,9 +301,74 @@ def open(
             name="issues", interval=timedelta(minutes=5)
         ),
         schema_inference=True,
+        disable=False,
     )
 
 
+def issue_child_resources(
+        log: Logger, http: HTTPMixin, config: EndpointConfig, timezone: ZoneInfo
+) -> list[common.Resource]:
+
+    def open(
+        stream: type[IssueChildStream],
+        binding: CaptureBinding[ResourceConfig],
+        binding_index: int,
+        state: ResourceState,
+        task: Task,
+        all_bindings,
+    ):
+        common.open_binding(
+            binding,
+            binding_index,
+            state,
+            task,
+            fetch_changes=functools.partial(
+                fetch_issues_child_resources,
+                http,
+                config.domain,
+                timezone,
+                config.advanced.projects,
+                stream,
+            ),
+            fetch_page=functools.partial(
+                backfill_issues_child_resources,
+                http,
+                config.domain,
+                timezone,
+                config.advanced.projects,
+                stream,
+            )
+        )
+
+    # Shift the cutoff back ISSUE_JQL_SEARCH_LAG duration to ensure backfills
+    # always cover ranges where Jira's API returns consistent results.
+    cutoff = datetime.now(tz=timezone) - ISSUE_JQL_SEARCH_LAG
```

Comment:
I  may be misunderstanding, but is this the same thing as `eventual consistency`? If so, would it be worth considering a `lookback` and `realtime` resource binding similar to `source-sage-intacct`?

Reply:
There are many streams already, so maybe this isn't as feasible for this connector anyway.

Reply:
Thinking about this again... The API rate limit being a concern would make this `lookback`/`realtime` approach not that great since it would cause more requests to be fired off.

Reply:
Gotcha! Yep, that sounds good. I think that's a good plan